linodecli/api_request.py

```diff
@@ -283,6 +283,9 @@ def _print_request_debug_info(method, url, headers, body):
     """
     print(f"> {method.__name__.upper()} {url}", file=sys.stderr)
     for k, v in headers.items():
+        # If this is the Authorization header, sanitize the token
+        if k.lower() == "authorization":
+            v = "Bearer *******************************"
         print(f"> {k}: {v}", file=sys.stderr)
```

Comment:
Just to make this behavior a bit more clear, could we introduce a new local variable here rather than mutating `v`?